tests/test_cli.py

```diff
@@ -868,3 +868,22 @@ def test_invoke_invalid_command(runner, tmpdir):
     assert (
         'Error: Invalid setting for [main][default_command]' in result.output
     )
+
+
+def test_show_priority(tmpdir, runner, create):
+    create(
+        'test.ics',
+        'SUMMARY:harhar\n'
+        'Priority:high\n'
```

Comment:
Looks like GitHub ate my previous comment.

Priority in files is numeric; should be `1` in this case. You should really use the `todo_factory` helper in this case anyway.